src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/CreateCertificate/AddCertificateTableViewCell.swift

```diff
@@ -0,0 +1,67 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+
+class AddCertificateTableViewCell: UITableViewCell, ReuseIdentifierProviding {
```

Comment:
Do we need a copy or could we also use the existing cell?

Reply:
Good idea, I will have a look if I can refactor that, too.